.github/workflows/e2e_regression.yml

```diff
@@ -24,20 +24,41 @@ env:
 
 jobs:
   regression-test:
-    runs-on: ubuntu-22.04
     permissions:
       contents: read
       packages: write
     strategy:
       matrix:
+        platform:
+          - name: AKS-CLH-SNP
+            runner: ubuntu-22.04
+            self-hosted: false
+          - name: K3s-QEMU-SNP
+            runner: SNP
+            self-hosted: true
+          - name: K3s-QEMU-TDX
+            runner: TDX
+            self-hosted: true
         case:
           - getdents
           - genpolicy
           - regression
+        exclude:
+          # getdents is a regression test for tardev-snapshotter
+          - platform:
+              self-hosted: true
```

Comment:
I'm surprised that this works, I always assumed it wouldn't.

Reply:
You mean the partial matching of objects? I did not know either - could be better documented, I guess.